### PR TITLE
Fix regression in SAM run configs with file input

### PR DESCRIPTION
**Pull request by rli**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Related Issue(s)
#2756 
#2758?
#2760 
#2762 

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt [hunk 1]
```diff
@@ -112,6 +114,26 @@ abstract class DotnetLocalLambdaRunConfigurationIntegrationTestBase(private val
             .containsEntry("AWS_SESSION_TOKEN", "")
     }
 
+    @Test
+    fun samIsExecutedWithFileInput() {
+        val input = Files.createTempFile(tempTestDirectory.toPath(), "tmp", null).also {
```
Comment (abrooksv) [inline jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt:119]: Why is this file created differently?

Comment (rli) [reply jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt:119]: these tests use rider's test base which don't have a fixture

Comment (abrooksv) [reply jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt:119]: Ah, fair

**Review by abrooksv — APPROVED** [review]

**Pull request merged**